docker/Dockerfile

```diff
@@ -11,12 +11,15 @@ ENV CI true
 ENV TRAVIS true
 ARG TRAVIS_BRANCH
 ARG TRAVIS_JOB_NUMBER
-ARG TRAVIS_PULL_REQUEST 
+ARG TRAVIS_PULL_REQUEST
 ARG TRAVIS_JOB_ID
 ARG TRAVIS_TAG
 ARG TRAVIS_REPO_SLUG
+ARG CODESTYLE
 ARG TRAVIS_COMMIT
+ARG TRAVIS_COMMIT_RANGE
 
+RUN [[ ${CODESTYLE} = "ON" ]] && exit 1
```

Comment:
It there are chance to put everything one dockerfile?

Just reorder the lines below and `exit` before `ccache` and the build.

Reply:
I set a single dockerfile. If everything is fine it will continue to build and run the tests.